packages/cyberstorm/src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -9,6 +9,10 @@ type BreadCrumbsProps = PropsWithChildren<{
   excludeHome?: boolean;
 }>;
 
+type BreadCrumbProps = PropsWithChildren<{
+  key?: number;
```

Comment:
Is this React's key-prop? Should it also support strings? And should it actually be used in `BreadCrumb`?

Reply:
What I meant was that the key, if given, is not actually passed to the span on [L38](https://github.com/thunderstore-io/thunderstore-ui/pull/655/files/f34b4dd648b69d4aff7a5aaf5044711d69c0eed1#diff-389a5ab4e4f2ec027c58d319f053029ac8a24b8227c218f4689ecff407e749d0R38). So when I use this component and pass the key, it gets yeeted and I get odd errors about not providing a key, although I did provide one.

Now that I think of it, React's types probably provide something that already provides the key, and maybe we should use that instead of defining it manually. Can't recall what that type might be from the top of my mind.

Reply:
I think I followed what you meant 🤔 I've added the key to the span 👍

Reply:
key is a default prop. You don't need to define it in the component's props, just pass it to the component. I fixed this.